prime-dotnet-webapi/Controllers/LookupsController.cs

```diff
@@ -14,8 +11,9 @@ namespace Prime.Controllers
     [Produces("application/json")]
     [Route("api/[controller]")]
     [ApiController]
-    //User needs at least the ADMIN or ENROLMENT role to use this controller
-    //[Authorize(Roles = PrimeConstants.PRIME_ADMIN_ROLE + "," + PrimeConstants.PRIME_ENROLMENT_ROLE)]
+    // User needs at least the ADMIN or ENROLMENT role to use this controller
+    // FIXME - add this back once there are OAuth tokens
```

Comment:
Can we communicate with the team that we're using FIXME as something to look for?

Reply:
I can change these to TODO, as a standard.